README.md

```diff
@@ -83,6 +83,71 @@ jobs:
 | 1   | **exit_code**  | Integer       | The exit code indicating the success or failure of the GitHub Actions workflow. |
 | 2   | **status**     |'failure' \| 'success'| The status of the GitHub Actions workflow, indicating success or failure. |
 
+
+## CLI (Command Line Interface)
+
+### Help Example
```

Comment:
suggestion: Let's begin with 'Usage' instead of 'Help Example' (replacing 'Usage Examples' with 'Examples')

----
### Usage
```
commitlint [-h] [-V] [--file FILE] [--hash HASH] [--from-hash FROM_HASH] [--to-hash TO_HASH] [--skip-detail] [commit_message]

...
```